### PR TITLE
Add css linter

### PR DESCRIPTION
**Pull request by marvinhagemeister**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-	"devDependencies": {
-		"async": "*",
-		"eslint": "*",
-		"jshint": "*",
-		"mocha": "*",
-		"parse-appcache-manifest": "*",
-		"ws": "*"
-	},
-	"private": true
+  "name": "bup",
```
Comment (phihag) [inline package.json]: Why the change in indentation here?

**Comment by phihag** [comment]
Thanks for the changes! Oftentimes, I really do like the ability to write multiple similar properties in a single line, but in the interest of consistency, I'll apply these changes.

Further modifications will be done in-tree.

File: package.json (hunk 1 above)
Comment (marvinhagemeister) [inline package.json]: That was npm's fault. Whenever `package.json` is touched by npm (even with `npm install`),  npm changes the indentation to two spaces. Should have changed the indentation back to tabs before committing...

**Comment by marvinhagemeister** [comment]
Yeah, I was unsure weather I should disable that rule as well. For short selectors writing them in one line totally makes sense.
